modules/rostests/winetests/comctl32/treeview.c

```diff
@@ -2724,6 +2724,11 @@ static void test_right_click(void)
     LRESULT result;
     POINT pt;
 
+#ifdef __REACTOS__
+    win_skip("test_right_click(), until CORE-14975 is fixed upstream and WINESYNC\n");
```

Comment:
Guard this by `!winetest_interactive` please

Reply:
Thanks for the reminder! Done.